Library/Homebrew/software_spec.rb

```diff
@@ -345,15 +345,15 @@ def select_download_strategy(specs)
 class BottleSpecification
   extend T::Sig
 
-  attr_rw :prefix, :cellar, :rebuild
+  attr_rw :prefix, :rebuild
   attr_accessor :tap
-  attr_reader :checksum, :collector, :root_url_specs, :repository
+  attr_reader :default_cellar, :checksum, :collector, :root_url_specs, :repository
 
   sig { void }
   def initialize
     @rebuild = 0
     @prefix = Homebrew::DEFAULT_PREFIX
-    @cellar = Homebrew::DEFAULT_CELLAR
+    @default_cellar = Homebrew::DEFAULT_CELLAR
```

Comment:
```suggestion
    @block_cellar = Homebrew::DEFAULT_CELLAR
```

or `@all_os_cellar` or something? `default` feels wrong here given it's being set below. I think it could even stay as `@cellar` and just use `tag_cellar` or something for the other.

Reply:
There needs to be a `cellar` method or @cellar attribute, to be backward compatible. I used `all_tags_cellar` if that's ok for you.